test/unit/rules/lint-aria-unsupported-elements-test.js

```diff
@@ -0,0 +1,78 @@
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'aria-unsupported-elements',
+
+  config: true,
+
+  good: ['<div role="button"></div>', '<img role="option">'],
+
+  bad: [
+    {
+      template: '<meta role>',
```

Comment:
Lets make the tests as representative as possible, I don't think `role` attribute without a value is a valid thing, is it?

Reply:
fixed